File: app/src/main/java/io/github/ovso/righttoknow/ui/childabuse/ChildAbuseActivity.java
package io.github.ovso.righttoknow.ui.childabuse;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.MenuItem;
import android.webkit.WebChromeClient;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import butterknife.BindView;
import io.github.ovso.righttoknow.R;
import io.github.ovso.righttoknow.framework.AdsActivity;

public class ChildAbuseActivity extends AdsActivity {
  @BindView(R.id.webview) WebView webView;

  @Override protected int getLayoutResId() {
    return R.layout.activity_child_abuse;
  }

  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    setTitle(R.string.child_abuse);
    WebSettings settings = webView.getSettings();
    settings.setJavaScriptEnabled(true);
    settings.setBuiltInZoomControls(true);
    settings.setDisplayZoomControls(true);
    settings.setLoadWithOverviewMode(true);
    settings.setUseWideViewPort(true);
    settings.setSupportZoom(true);
    webView.setWebViewClient(new WebViewClient());
    webView.setWebChromeClient(new WebChromeClient());
    webView.loadUrl(
        "http://m.post.naver.com/viewer/postView.nhn?volumeNo=9367173&memberNo=22718804&vType=VERTICAL");
  }

  @Override public boolean onOptionsItemSelected(MenuItem item) {
    finish();
    return super.onOptionsItemSelected(item);
  }
}

File: app/src/main/java/io/github/ovso/righttoknow/ui/childabuse/ChildAbuseActivity.kt
package io.github.ovso.righttoknow.ui.childabuse

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.MenuItem
import android.webkit.WebChromeClient
import android.webkit.WebViewClient
import io.github.ovso.righttoknow.R
import io.github.ovso.righttoknow.databinding.ActivityChildAbuseBinding
import io.github.ovso.righttoknow.exts.viewBinding
import io.github.ovso.righttoknow.framework.AdsActivity
import kotlinx.android.synthetic.main.content_abuse.*

class ChildAbuseActivity : AdsActivity() {

  private val binding by viewBinding(ActivityChildAbuseBinding::inflate)

  @SuppressLint("SetJavaScriptEnabled")
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(binding.root)
    setSupportActionBar(toolbar)
    supportActionBar?.setDisplayHomeAsUpEnabled(true)
    setTitle(R.string.child_abuse)
    val settings = webview.settings
    settings.javaScriptEnabled = true
    settings.builtInZoomControls = true
    settings.displayZoomControls = true
    settings.loadWithOverviewMode = true
    settings.useWideViewPort = true
    settings.setSupportZoom(true)
    webview.webViewClient = WebViewClient()
    webview.webChromeClient = WebChromeClient()
    webview.loadUrl(
      "http://m.post.naver.com/viewer/postView.nhn?volumeNo=9367173&memberNo=22718804&vType=VERTICAL"
    )
  }

  override fun onOptionsItemSelected(item: MenuItem): Boolean {
    finish()
    return super.onOptionsItemSelected(item)
  }
}
